### ISSUE TITLE
Plans on supporting ILCE-6100?

### ISSUE THREAD

**Issue by juansa91** [ma1co/Sony-PMCA-RE#443]
Hey! Amazing project first of all. 
I bought an a6100 but i cant change the language, when using the pmca-gui in the tweaks section i get: Unknown device: ILCE-6100
Thanks!

**Comment by juansa91** [comment]
I got it to work on 18-22
Thank you very much!

**Comment by FAR18-droid** [comment]
> I got it to work on 18-22
> Thank you very much!

Hi, 
I have sony A6100 and has the same problem. Could you tell me where I can download the 18-22 version? Thank you in advance @juansa91

**Comment by juansa91** [comment]
Sure friend, here it is: 
https://github.com/evianita08/pmca_v18.22

I used the console version btw

**Comment by FAR18-droid** [comment]
Thank you for the link. I already try the v18.22 but gui version.
Do you use the zadig too? @juansa91

**Comment by juansa91** [comment]
> Thank you for the link. I already try the v18.22 but gui version. Do you use the zadig too? @juansa91

Yeah, I used zadig to install libusb-win32. You have to install it first, then run pmca serviceshell and after that install it again, that's what worked for me.

**Comment by FAR18-droid** [comment]
Oh is it ok install the libusb driver? Because the original driver was different I think.

**Comment by FAR18-droid** [comment]
> > Thank you for the link. I already try the v18.22 but gui version. Do you use the zadig too? @juansa91
> 
> Yeah, I used zadig to install libusb-win32. You have to install it first, then run pmca serviceshell and after that install it again, that's what worked for me.

Sorry, which one is pmca serviceshell? @juansa91

**Comment by juansa91** [comment]
> > > Thank you for the link. I already try the v18.22 but gui version. Do you use the zadig too? @juansa91
> > 
> > 
> > Yeah, I used zadig to install libusb-win32. You have to install it first, then run pmca serviceshell and after that install it again, that's what worked for me.
> 
> Sorry, which one is pmca serviceshell? @juansa91

Pmca can be the GUI or the console version, if u use the console version you have to execute it via CMD or powershell and do "pmca-console.exe serviceshell" and then you can access the tweaks. 
See ma1co docs

**Comment by FAR18-droid** [comment]
Finally I can unlock the language on ILCE-6100 using the v18.22 gui version. Even it always restarting when I try to scroll of language menu especially until Indonesia, but at least it works perfectly in english.
I'm not sure what am doing but I've failed for about 4 times before it works. Thank you very much for your help @juansa91 
And of course thank you @ma1co for this amazing project !!

**Comment by rafahub11** [comment]
> > > > Thank you for the link. I already try the v18.22 but gui version. Do you use the zadig too? @juansa91
> > > 
> > > 
> > > Yeah, I used zadig to install libusb-win32. You have to install it first, then run pmca serviceshell and after that install it again, that's what worked for me.
> > 
> > 
> > Sorry, which one is pmca serviceshell? @juansa91
> 
> Pmca can be the GUI or the console version, if u use the console version you have to execute it via CMD or powershell and do "pmca-console.exe serviceshell" and then you can access the tweaks. See ma1co docs

Hi, I'm having the same issue. I installed the GUI and the console version. The GUI opens on windows but it does not find the camera, as far as I understand.
Then I installed the zadig to install libusb-win32. But I don't understand how this works. I think I ended up installing the libusb-win32 without the need of the zadig.

After all I tried opening the console version but it simply tries running the app, a "cmd" window opens and automatically closes right after. So I can't really open the console version as well...

Any clue on what is happening? Am I doing something wrong?

_Post edition below:_

I acctually think I got the thing about installing the libusb-win32 on the camera (screenshot). Is that correct?
![Screenshot 2023-10-19 193456](https://github.com/ma1co/Sony-PMCA-RE/assets/148504562/4a7117d8-7cb9-44f5-b486-7aa9389f80e8)
